viral/learning_stages.py

```
@@ -72,28 +73,58 @@
 
+# 26 is ok, 27 dont use,
+# 30 is ok 33 is good 31 is ok not great
 
-def get_session(
-    mouse_name: str, date: str, metadata: pd.DataFrame, stage: str
-) -> Cached2pSession:
 
-    row = metadata[metadata["Date"] == date].squeeze(axis=0)
-    session_type = row["Type"].lower()
-    assert "learning" in session_type if stage == "learned" else stage in session_type
+def oasis_runner() -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)